content/millennium/r4/financial/charge-item.md

```diff
@@ -0,0 +1,113 @@
+* [Entered Date](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.enteredDate){:target="_blank"}
+* [Reason](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason){:target="_blank"}
+* [Account](https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.account){:target="_blank"}
+* [Extensions including custom attribute, description, modifier, net price, offset by, performing location, procedure, revenue code, and unit price](#extensions){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:charge_item, :r4) %>
+
+## Extensions
+
+* [Custom Attribute]
+* [Description]
+* [Modifier]
+* [Net Price]
+* [Offset By]
+* [Performing Location]
+* [Procedure]
+* [Revenue Code]
+* [Unit Price]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                     | Value\[x] Type                    | Description
+------------------------|-----------------------------------|--------------------------------------------------------------------------
+ `custom-attribute`     | None (contains nested extensions) | A client defined custom attribute for the resource. Attribute values can be of type [`integer`], [`string`], [`decimal`], or [`date`].
+ `discription`          | [`string`]                        | A description providing additional details of the resource.
+ `modifier`             | None (contains nested extensions) | A code providing additional detail about a product or service.
+ `net-price`            | [`Money`]                         | The quantity times the unit price for a resource (total price).
+ `offset-by`            | [`Reference`]                     | Indicates a resource that this resource is offset by. This resource is no longer active when offset.
+ `performing-location`  | [`Reference`]                     | A location where the resource was performed.
+ `procedure`            | None (contains nested extensions) | Procedure performed on the patient associated to the resource.
+ `revenue-code`         | None (contains nested extensions) | The type of revenue or cost center providing the product and/or service.
+ `unit-price`           | [`Money`]                         | The price of a single unit for the resource.
+
+## Retrieve by id
+
+List an individual ChargeItem by its id:
+
+    GET /ChargeItem/:id
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/ChargeItem/157320807
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_charge_item_read) %>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+[`CodeableConcept`]: http://hl7.org/fhir/r4/datatypes.html#CodeableConcept
```

Comment:
http -> https

Reply:
this was not being used. Removed [here](https://github.com/cerner/fhir.cerner.com/pull/605/commits/61aa81539c2b29d67f02a8bb87769033ab4cb6bb#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbL95).
Money and reference are still http URL